opentelemetry-sdk/CHANGELOG.md

```diff
@@ -263,6 +263,13 @@ hardcoded limit of 2000 and no ability to change it. This feature will be
 re-introduced in a future date, along with the ability to change the cardinality
 limit.
 
+- Refactor modules. This is *Breaking* change, if you author custom
```

Comment:
nit - we normally start with `**Breaking**` as first word to indicate breaking :)